commons/commons-bio/src/main/kotlin/ebi/ac/uk/model/extensions/SubExt.kt

```diff
@@ -39,12 +78,13 @@ var Submission.rootPath: String?
  * Obtain the submission title attribute if present.
  */
 var Submission.title: String?
-    get() = this[SubFields.TITLE]
+    get() = find(SubFields.TITLE)
     set(value) {
         value?.let { this[SubFields.TITLE] = it }
     }
 
+fun Submission.addAccessTag(accessTag: String) = accessTags.add(accessTag)
 fun Submission.allFiles() = section.allFiles() + section.allSections().flatMap { it.allFiles() }
-fun Submission.getSectionByType(name: String) = section.allSections().first { it.type == name }
-fun Submission.fileListSections() = (section.allSections() + section).filterNot { it.fileListName.isNullOrBlank() }
+fun Submission.getSectionByType(name: String): Section = section.allSections().first { it.type == name }
+fun Submission.fileListSections() = (section.allSections() + section).filterNot { it.fileListName == null }
```

Comment:
For this case, if we had an empty file list it shouldn't be considered a `fileListSection`, that's why it was `isNullOrBlank` before

Reply:
if the `fileListName` is present but empty this is an error and should be trigger or system should break. it should not be possible to have an empty `fileListName`